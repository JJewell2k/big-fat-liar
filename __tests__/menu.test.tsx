import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import App from '../pages/_components/MenuBar/_components/Menu/index'

describe('MenuBar -> Menu component', () => {
  let app: ShallowWrapper

  beforeEach(() => {
    app = shallow(<App items={[]} title="Apple" />)
  })

  it('Should renders properly', () => {
    expect(shallowToJson(app)).toMatchSnapshot()
  })
})
