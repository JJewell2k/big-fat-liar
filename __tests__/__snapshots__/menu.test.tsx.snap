// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuBar -> Menu component Should renders properly 1`] = `
<div
  className="absolute bg-gray-700 bg-opacity-50 p-1 rounded-md left-0 border border-gray-500 border-box top-5"
/>
`;
